### ISSUE TITLE
Include doesn't work

### ISSUE THREAD

**Issue by Lughino** [douglasduteil/isparta#56]
I tried to scan `.es6` files source with `include` options but seem not work!
If replace my source code in `.js` it works.
This is the command:
```shell
./node_modules/.bin/babel-node ./node_modules/.bin/isparta cover --include "**/*.es6" -v ./node_modules/.bin/_mocha -- "./test/**/*.es6" --compilers js:babel/register --require "./test/index.js"
```

This is the verbose output:
```shell
Using configuration
-------------------
verbose: true
instrumentation:
    root: .
    extensions:
        - .js
    default-excludes: true
    excludes: []
    embed-source: false
    variable: __coverage__
    compact: true
    preserve-comments: false
    complete-copy: false
    save-baseline: false
    baseline-file: ./coverage/coverage-baseline.json
    include-all-sources: false
    include-pid: false
    preload-sources: false
reporting:
    print: summary
    reports:
        - lcov
    dir: ./coverage
    watermarks:
        statements: [50, 80]
        lines: [50, 80]
        functions: [50, 80]
        branches: [50, 80]
    report-config:
        clover: {file: clover.xml}
        cobertura: {file: cobertura-coverage.xml}
        json: {file: coverage-final.json}
        json-summary: {file: coverage-summary.json}
        lcovonly: {file: lcov.info}
        teamcity: {file: null}
        text: {file: null, maxCols: 0}
        text-lcov: {file: lcov.info}
        text-summary: {file: null}
hooks:
    hook-run-in-context: false
    post-require-hook: null
    handle-sigint: false
check:
    global:
        statements: 0
        lines: 0
        branches: 0
        functions: 0
        excludes: []
    each:
        statements: 0
        lines: 0
        branches: 0
        functions: 0
        excludes: []

-------------------

Isparta options : 
  { '0': 'cover',
  '--': 
   [ './test/**/*.es6',
     '--compilers',
     'js:babel/register',
     '--require',
     './test/index.js' ],
  include: [ '**/*.es6', './node_modules/.bin/_mocha' ],
  verbose: true,
  cmd: './node_modules/.bin/_mocha',
  _: [ 'cover', './node_modules/.bin/_mocha' ],
  report: 'lcv' }
Running: node /Users/lug/Projects/pr221/node_modules/.bin/_mocha ./test/**/*.es6 --compilers js:babel/register --require ./test/index.js
```

**Comment by yhsiang** [comment]
I sent a PR #61 for this.

**Comment by douglasduteil** [comment]
Closes with #61
